dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/GenericFilter.java

```diff
@@ -199,7 +199,7 @@ private Object[] getGsonGenericArgs(final Object[] args, Type[] types) {
             }
             String str = args[i].toString();
             try {
-                return GsonUtils.fromJson(str, types[i]);
```

Comment:
Some gson related annotation may failed.

Reply:
> Some gson related annotation may failed.

I made a mistake.
Thanks for your guidance